base/docs/basedocs.jl

```diff
@@ -142,6 +142,134 @@ keywords[:return] = doc"""
   block) and gives a value back to `map`. `test2([5,6,7])` then returns `[5,12,7]`.
   """
 
+keywords[:if] = doc"""
+  `if` performs conditional evaluation, which allows portions of code to be evaluated or not evaluated depending on the value of a boolean expression.
+  Here is the anatomy of the `if`-`elseif`-`else` conditional syntax:
+
+	if x < y
+	  println("x is less than y")
+	elseif x > y
+	  println("x is greater than y")
+	else
+	  println("x is equal to y")
+	end
+
+  If the condition expression `x < y` is true, then the corresponding block is evaluated;
+  otherwise the condition expression `x > y` is evaluated, and if it is true, the corresponding block is evaluated; if neither expression is true, the `else` block is evaluated.
+  The `elseif` and `else` blocks are optional, and as many `elseif` blocks as desired can be used.
+  """
+
+keywords[:elseif] = doc"""
+  `elseif` may optionally be used in conditional evaluation to execute a block of code when neither the `if` expression
+  nor any preceding `elseif` expression returned true. Here is the anatomy of the `if`-`elseif`-`else` conditional syntax:
+
+	if x < y
+	  println("x is less than y")
+	elseif x > y
+	  println("x is greater than y")
+	else
+	  println("x is equal to y")
+	end
+
+  The `elseif` and `else` blocks are optional, and as many `elseif` blocks as desired can be used.
+  """
+
+keywords[:else] = doc"""
+  `else` may optionally be used in conditional evaluation to execute a block of code when neither the `if` expression nor any `elseif` expression returned true.
+  Here is the anatomy of the `if`-`elseif`-`else` conditional syntax:
+
+	if x < y
+	  println("x is less than y")
+	elseif x > y
+	  println("x is greater than y")
+	else
+	  println("x is equal to y")
+	end
+
+  The `else` and `elseif` blocks are optional.
+  """
+
+keywords[:for] = doc"""
+  `for` loops repeatedly evaluate the body of the loop by iterating over a sequence of values.  For example:
+
+        for i in [1,4,0]
+          println(i)
+        end
+  """
+
+keywords[:while] = doc"""
+  `while` loops repeatedly evaluate a conditional expression, and continues evaluating the body of the while loop so long as
+  the expression remains `true`. If the condition expression is false when the while loop is first reached, the body is never evaluated.
+  For example:
+
+        while i <= 5
+          println(i)
+          i += 1
+        end
+  """
+
+keywords[:end] = doc"""
+  `end` marks the conclusion of a block of expressions.
+  In the example below, `end` marks the conclusion of a `function`.
+
+        function foo()
+          println("hello, world")
+         end
+
+  `end` marks the conclusion of all kinds of expression blocks: `module`, `type`, `begin`, `let`, `for`, etc.
+  """
+
+keywords[:try] = doc"""
```

Comment:
As with the `if`-`elseif`-`else` comment above, could the `try` and `catch` docs be joined together?